Source/WebCore/crypto/CryptoAlgorithmIdentifier.h

```diff
@@ -27,6 +27,7 @@
 
 namespace WebCore {
 
+constexpr auto sha224DeprecationMessage = "SHA224 is not supported";
```

Comment:
Should be using the `_s` operator here instead?  I guess if this is `constexpr`, it's probably equivalent.

```suggestion
auto sha224DeprecationMessage = "SHA224 is not supported"_s;
```

Reply:
that fails to compile with "....must be initialized by a constant expression"